docs/Extensions.md

```diff
@@ -0,0 +1,50 @@
+# fastlane extensions
+
+### Local actions
+
+You can create your own actions to extend the functionality of `fastlane`. The action you create will behave exactly like the built in actions.
+
+Just run `fastlane new_action`. Then enter the name of the action and edit the generated Ruby file in `fastlane/actions/[action_name].rb`.
+
+From then on, you can just start using your action in your `Fastfile`.
+
+### Submitting the action to the fastlane main repo
+
+If you think your extension will be useful to other developers as well, clone fastlane, copy your local action to `fastlane/lib/fastlane/actions` and submit a pull request.
+
+### Remote plugins
```

Comment:
I don't see why we need to add yet another word (Extension).

So maybe call this file Actions,md instead of Extensions.md and rename the other Actions.md into BundledActions.md.